Framework/Email.php

```diff
@@ -144,16 +144,15 @@ public function sendEmailToSpaceMembers($params, $lang = "", $mailing=null) {
         $fromName = "Platform-Manager";
         $subject = CoreTranslator::MailSubjectPrefix($spaceName, $lang) . " " . $subject;
         $mailerSetCopyToFrom = $this->getMailerSetCopyToFrom($spaceId);
-
         // get the emails
         switch ($params["to"]) {
             case "all":
-                $toAddress =
-                    $this->formatAddresses($modelSpace->getEmailsSpaceActiveUsers($spaceId));
+                $toAddress = $this->formatAddresses($modelSpace->getEmailsSpaceActiveUsers($spaceId));
                 break;
             case "managers":
-                $toAddress =
-                    $this->formatAddresses($modelSpace->getEmailsSpaceManagers($spaceId));
+                $toAddress = $this->formatAddresses($modelSpace->getEmailsSpaceManagers($spaceId));
+                $modelUser = new CoreUser();
+                $params['content'] = "From " . $modelUser->getUserFUllName($params['id_user']) . " :</br>" . $params['content'];
```

Comment:
why customizing content here ? it is previous function setting the message content that should add the user info if necessary